lib/oat/adapters/json_api.rb

```diff
@@ -80,8 +80,9 @@ def entity(name, obj, serializer_class = nil, context_options = {}, &block)
         end
       end
 
-      def entities(name, collection, serializer_class = nil, context_options = {}, &block)
+      def entities(name, collection, serializer_class = nil, context_options = nil, &block)
         return if collection.nil? || collection.empty?
+        context_options ||= {}
```

Comment:
The default only takes effect when the arg is not supplied. The intention is for `context_options` to be `{}` even if the argument is passed in explicitly as `nil`.